aws/resource_aws_spot_fleet_request.go

```diff
@@ -309,9 +310,93 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 						},
 					},
 				},
-				Set: hashLaunchSpecification,
+				Set:          hashLaunchSpecification,
+				AtLeastOneOf: []string{"launch_specification", "launch_template_config"},
+			},
+			"launch_template_config": {
+				Type:          schema.TypeSet,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"launch_specification"},
+				AtLeastOneOf:  []string{"launch_specification", "launch_template_config"},
```

Comment:
😅 Now seeing this together, I believe this is semantically the same as declaring this as (on both `launch_specification` and `launch_template_config`):

```
ExactlyOneOf: []string{"launch_specification", "launch_template_config"},
```

Sorry about that. I'll create an feature request in tfproviderlint to suggest that.

Reply:
just to make sure, im swapping both of these lines with:
```
ExactlyOneOf: []string{"launch_specification", "launch_template_config"},
```